platform-common-testing/src/main/java/com/softicar/platform/common/testing/Asserts.java

```diff
@@ -71,11 +71,28 @@ public static void assertExceptionMessage(IDisplayString expectedMessage, INulla
 		try {
 			thrower.apply();
 			fail("An expected exception failed to occur.");
+		} catch (NullPointerException exception) {
```

Comment:
An NPE has `exception.getMessage() == null`. This could result in confisuing behavior of this assertion method. Hence the new distinct handling of NPE.